tree.go

```diff
@@ -16,11 +16,12 @@ type Tree struct {
 	size    int
 	order   []string
 	cid     cid.Cid
+	rawdata []byte
```

Comment:
Appears to bu unused / not implemented here

Reply:
Whoops, fixed.